src/main/java/com/auth0/client/auth/PasswordlessRealm.java

```diff
@@ -0,0 +1,32 @@
+package com.auth0.client.auth;
+
+/**
+ * Represents the type of realm used for Passwordless authentication.
+ */
+public enum PasswordlessRealm {
```

Comment:
the only reason to not use enums is that they can't be instantiated outside this class, can they? so there's no support for custom passwordless connections. 

Ideally, we could have this good defaults and then let the developer do something like this

```java
new PasswordlessRealm("sms-custom");
```

Reply:
Right, enums don't really support the notion of being instantiated - they represent a fixed set of values. Since an enum wouldn't support custom passwordless connections, I'll update to use a String param in the API and remove this enum.